### ISSUE TITLE
Adding a basic theme to templates

### ISSUE THREAD

**Issue by Germoe** [forestryio/create-static-site#49]
Adding a theme to C-S-S is quite painful at the moment. I believe Hugo, Jekyll etc. should be installed with a basic theme attached to it to reduce configuration headaches.

If I use the install theme routine lined out in Hugo docs I can install a theme but it won't show when I run `npm start`, it doesn't throw any errors it will just ignore completely that I have installed a theme.

I believe Create-Static-Site should be for everyone - including beginners
If a configuration is left out, beginners will be prevented from using C-S-S or at least it will cause them major headaches. If someone that is an advanced user comes across an already installed theme I imagine it to be a minor annoyance but if the C-S-S brings enough value these users will stick around.

**Comment by jamespohalloran** [comment]
Agreed @Germoe. I think the default layout content should be moved to a theme, so that you can easily switch themes and not have to delete the layout directory.
